examples/renderable/main.rs

```diff
@@ -184,10 +185,10 @@ impl SimpleState for Example {
     }
 }
 
-fn main() -> Result<(), Error> {
+fn main() {
     amethyst::start_logger(Default::default());
 
-    let app_root = application_root_dir()?;
+    let app_root = application_root_dir().expect("Could not create application root");
```

Comment:
Why?

Reply:
Because of `EventLoop::run() -> !`

Reply:
Doesn't prevent you from returning a result.

Reply:
calling a function that does not return will yield unreachable code after that.
See here: https://repl.it/repls/UntimelyWelcomePatterns

Without allowing unreachable code, it is not possible to have main return anything. If I am correct.

Reply:
It can still return `Result::Err`, just not `Result::Ok`. The compiler is warning you about unreachable code because `Ok(..)` can never be reached. Removing that gets rid of the warning.
https://repl.it/repls/LavenderAnguishedWordprocessor

Reply:
Ahh. Thanks. I will update that than back to a Result.